src/main/scala/me/rexim/morganey/module/ModuleFinder.scala

```diff
@@ -38,6 +40,16 @@ class ModuleFinder(val paths: List[File]) {
     Option(resourceUrl)
   }
 
+  def findAllModulesInClasspath(): Set[String] = {
```

Comment:
I think it's better to rename it to something like `findAllModulesInIndex` to emphasize that it's not the only way of discovering modules. It is also possible to access a module if you know its full path, even if such module is not registered in the Morganey index.